### PR TITLE
Adnuntius Bid Adapter: bugfix to set undefined if no bids

### PR DESCRIPTION
**Pull request by mikael-lundin**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Fixin an issue that will set undefined on bid response if there are no bids.

- contact email of the adapter’s maintainer: mikael@adnuntius.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

### CODE REVIEW

File: modules/adnuntiusBidAdapter.js [hunk 1]
```diff
@@ -68,10 +68,10 @@ export const spec = {
       } else return response
     }, {});
 
-    const bidResponse = bidRequest.bid.map(bid => bid.bidId)
-      .reduce((request, adunitId) =>
-        request.concat(bidResponsesById[adunitId])
-        , []);
+    const bidResponse = bidRequest.bid.map(bid => bid.bidId).reduce((request, adunitId) => {
+      if (bidResponsesById[adunitId]) request.push(bidResponsesById[adunitId])
```
Comment (ChrisHuie) [inline modules/adnuntiusBidAdapter.js:72]: The request.push doesn't appear to be scoped to the if statment block. Am I looking at this wrong?

Comment (mikael-lundin) [reply modules/adnuntiusBidAdapter.js:72]: you are correct, I look at the bidResponsesById object if it contains the adunitId, and if it does I will push that object into the request array that will be returned to the auction as winners. this is just to make sure that the order of adunits will be correct.

Comment (mikael-lundin) [reply modules/adnuntiusBidAdapter.js:72]: I can make it a bit more clear what happens in the code if you wish?

Comment (ChrisHuie) [reply modules/adnuntiusBidAdapter.js:72]: I've honestly never seen an if statment in javascript without brackets so freaks me out. lol

Comment (mikael-lundin) [reply modules/adnuntiusBidAdapter.js:72]: Haha i can add brackets to it if you wish. :)

Comment (mikael-lundin) [reply modules/adnuntiusBidAdapter.js:72]: And no, nothing will change.

Comment (ChrisHuie) [reply modules/adnuntiusBidAdapter.js:72]: thanks 🙏

Comment (mikael-lundin) [reply modules/adnuntiusBidAdapter.js]: We're in business now. :P ( = should be fixed now)

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**